src/Components/Error/ErrorPage.tsx

```diff
@@ -0,0 +1,35 @@
+import { ErrorBoundaryPage as _ErrorBoundaryPage } from '@redhat-cloud-services/insights-common-typescript';
```

Comment:
I migrated this component to `frontend-components` under the name `ErrorBoundary`

I guess this is the code change needed, but didn't confirm. 
```suggestion
import { ErrorBoundary as _ErrorBoundaryPage } from '@redhat-cloud-services/frontend-components';
```

Reply:
`3.9.1` (or later if any) surelly has it

Reply:
OH! Correction, updated and found it!

Reply:
The first design seems cleaner than the new one in front-end components. Has design finalized their review of this yet? (Just curious).

Before: 
![Screen Shot 2022-06-01 at 1 49 36 PM](https://user-images.githubusercontent.com/38083295/171489541-5ae03541-c2d8-426d-b762-27bc57536a93.png)

After: 
![Screen Shot 2022-06-01 at 1 50 20 PM](https://user-images.githubusercontent.com/38083295/171489614-be44a2d7-a3f1-4104-8241-43b4d7e60ffc.png)

Reply:
Cool!